test/dynamo/test_logging.py

```diff
@@ -185,7 +185,11 @@ def test_dynamo_error(self, records):
 
 from user code:
    File "test_logging.py", line N, in dynamo_error_fn
-    output = output.add(torch.ones(10, 10))""",  # noqa: B950
+    output = output.add(torch.ones(10, 10))
+
+Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace. For even more developer context, set TORCH_LOGS="+dynamo"
```

Comment:
nit: `Set TORCHDYNAMO_VERBOSE=1 for the internal compiler stack trace (please do this especially if you're reporting a bug to PyTorch)`